templates/error.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <title>Build error</title>
+    <script src="/ember-cli-live-reload.js" type="text/javascript"></script>
```

Comment:
So the purpose of this is to get live-reload without requiring a browser extension?